### PR TITLE
Add EGI 2023 blog post

### PR DESCRIPTION
**Pull request by sanjaysrikakulam**

### CODE REVIEW

**Comment by bgruening** [comment]
Maybe a picture of the booth? and links to the talks?

**Comment by sanjaysrikakulam** [comment]
> Maybe a picture of the booth? and links to the talks?

Added a picture of the booth and the links to the slides in the table.

File: content/news/2023-07-24-egi2023/index.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
+title: 'Freiburg Galaxy team at the EGI 2023'
+date: "2023-07-24"
+tease: "From June 19th to 23rd, members from the Freiburg Galaxy team travelled to Poznań, Poland to participate and present the EuroScienceGateway (ESG) project at the EGI 2023"
+hide_tease: false
+subsites: [all-eu]
+main_subsite: eu
+---
+
+During 19th-23rd of June, members (Björn Grüning, Sebastian Schaaf, and Sanjay Kumar Srikakulam) of the European Galaxy team travelled to Poznań to join the EGI 2023 to present the [EuroScienceGateway project](https://galaxyproject.org/projects/esg/) along with collaborators (Sebastian Luna-Valero, Hakan Bayindir, Lukasz Dutka, Abdul Rahman Azab, and Maiken Pedersen) from different work packages of ESG. At EGI 2023, Galaxy EU for the first time held a booth; a dedicated session with multiple talks introducing and presenting Galaxy, and the ESG project.
```
Comment (sebastian-schaaf) [inline content/news/2023-07-24-egi2023/index.md:10]:
```suggestion
During 19th-23rd of June, Björn Grüning, Sebastian Schaaf, and Sanjay Kumar Srikakulam from the European Galaxy Team travelled to Poznań to join the EGI2023 conference and present the [EuroScienceGateway project](https://galaxyproject.org/projects/esg/) along with collaborators. With Sebastián Luna-Valero and Enol Fernández (European Grid Initiative/NL), Hakan Bayındır (ULAKBIM/TR), Łukasz Dutka (Cyfronet/PL), Abdulrahman Azab and Maiken Pedersen (University of Oslo/NO), work packages 1, 3 and 4 of ESG were represented. 

At EGI 2023, we first time held a booth on Galaxy and usegalaxy.eu, presenting the European Galaxy Server to a wider audience from various scientific fields. A dedicated conference session with multiple talks introduced Galaxy and presented its opportunities, as well as the ESG project.
```

File: content/news/2023-07-24-egi2023/index.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+## Day 2 - June 20
+
+On the second day, Sanjay, and Sebastian set up the ESG booth. In the evening Sanjay pitched the booth and advertised Galaxy and the ESG project during the **Exhibition opening and 1 minute pitches** session.
+
+![EGI 2023 booth picture](./EGI_2023_booth.jpg)
+
+## Day 4 - June 22
+
+The members of the ESG project held a dedicated one and half hour session **EuroScienceGateway: an open infrastructure for data-driven research** where they gave the following three talks followed by a Q&A.
```
Comment (sebastian-schaaf) [inline content/news/2023-07-24-egi2023/index.md:22]:
```suggestion
The members of the ESG project held a dedicated 90 minutes session **EuroScienceGateway: an open infrastructure for data-driven research**, where they gave the following three talks, followed by a Q&A.
```

File: content/news/2023-07-24-egi2023/index.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+| Presenters | Title | Links |
+|---|---|---|
+| Sebastian, and Sanjay | Introduction to Galaxy | [Slides](https://docs.google.com/presentation/d/e/2PACX-1vQfHWkU_rg2KaNe58Fp341Il_Y0UxMnQvPA1REs90WvfLfksNvQMcKzn2TVBnd8DQ/pub?start=true&loop=true&delayms=3000) |
+| Björn | Let's talk about storage Or Empower EOSC Users to Bring Their Own Storage (BYOS) | [Slides](https://docs.google.com/presentation/d/e/2PACX-1vQVTNtkCPMPGE53RGvqmitR__xEI0Rq0gZjwnLkWERNJ7B7_vuaFqOMF7KXDIXA4w/pub?start=true&loop=true&delayms=3000) |
+| Abulrahman, Sebastián, and Hakan | Bring Your Own Compute (BYOC) | [Slides](https://docs.google.com/presentation/d/e/2PACX-1vQg8bu0q3RomaQwQmN50saGHNsiRfzO0rg15fB3Nej9aFLF8X4uuLRDHdSeeb8z4Q/pub?start=true&loop=true&delayms=3000) |
```
Comment (sebastian-schaaf) [inline content/news/2023-07-24-egi2023/index.md:28]:
```suggestion
| Abdulrahman, Sebastián, and Hakan | Bring Your Own Compute (BYOC) | [Slides](https://docs.google.com/presentation/d/e/2PACX-1vQg8bu0q3RomaQwQmN50saGHNsiRfzO0rg15fB3Nej9aFLF8X4uuLRDHdSeeb8z4Q/pub?start=true&loop=true&delayms=3000) |
```

File: content/news/2023-07-24-egi2023/index.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+During the session, the members introduced Galaxy espeically the Galaxy EU's design, setup and architecture. Highlighted the recent developments, such as object stores and distributed computing (PULSAR) a core component of ESG and the efforts to mature it to TLR-9, enabling the efficient utilization of resroucres across Europe.
```
Comment (sebastian-schaaf) [inline content/news/2023-07-24-egi2023/index.md:32]:
```suggestion
During the session, the members introduced Galaxy,  in particular the Galaxy's EU instance's design, setup and architecture. Highlighted the recent developments, such as object stores and distributed computing. With [PULSAR ](https://pulsar-network.readthedocs.io/en/latest/index.html) being a core component of ESG (and a product of the Galaxy community), the efforts to mature it to TLR-9 and enabling the efficient utilization of resources across Europe were sketched.
```

File: content/news/2023-07-24-egi2023/index.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+The initiative aims to create an open infrastructure for data-driven research, focusing on providing a user-friendly scientific workflow and data integration platform. This endeavor seeks to achieve robustness, scalability, and integration, enabling federated computational infrastructures and democratizing research data analysis.
+
+During the session various aspects of ESG and Galaxy were discussed including the integration of OneData platform, exploring DIRAC as a new metascheduler and ARC as a new job-runner.
+
+# Thank You !
+
+The European Galaxy team would like to **thank** the **Galaxy community**, **members of the ESG** for all their contributions that were presented and **EGI** for the fantastic organisation of this meeting.
+
+![EGI 2023 participant picture](https://pbs.twimg.com/media/Fznr0-OWYAEJ85a?format=jpg&name=large)
```
Comment (sebastian-schaaf) [inline content/news/2023-07-24-egi2023/index.md]:
```suggestion
![EGI2023 participants picture](https://pbs.twimg.com/media/Fznr0-OWYAEJ85a?format=jpg&name=large)
```

File: content/news/2023-07-24-egi2023/index.md [hunk 6]
```diff
@@ -0,0 +1,40 @@
+---
+title: 'Freiburg Galaxy team at the EGI 2023'
+date: "2023-07-24"
+tease: "From June 19th to 23rd, members from the Freiburg Galaxy team travelled to Poznań, Poland to participate and present the EuroScienceGateway (ESG) project at the EGI 2023"
```
Comment (sebastian-schaaf) [inline content/news/2023-07-24-egi2023/index.md:4]:
```suggestion
tease: "From June 19th to 23rd, members from the Freiburg Galaxy team traveled to Poznań, Poland to participate and present the EuroScienceGateway (ESG) project at the EGI2023 meeting"
```

**Comment by sanjaysrikakulam** [comment]
Thank you @sebastian-schaaf! I have accepted all the suggestions and also update the images with the ones you sent. 

@bgruening This PR is now ready to be approved and merged.

File: content/news/2023-07-24-egi2023/index.md [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+date: "2023-07-24"
+tease: "From June 19th to 23rd, members from the Freiburg Galaxy team traveled to Poznań, Poland to participate and present the EuroScienceGateway (ESG) project at the EGI2023 meeting"
+hide_tease: false
+subsites: [all-eu]
+main_subsite: eu
+---
+
+During 19th-23rd of June, Björn Grüning, Sebastian Schaaf, and Sanjay Kumar Srikakulam from the European Galaxy Team travelled to Poznań to join the EGI2023 conference and presented the [EuroScienceGateway project](https://galaxyproject.org/projects/esg/) along with collaborators. With Sebastián Luna-Valero and Enol Fernández (European Grid Initiative/NL), Hakan Bayındır (ULAKBIM/TR), Łukasz Dutka (Cyfronet/PL), Abdulrahman Azab and Maiken Pedersen (University of Oslo/NO), work packages 1, 3 and 4 of ESG were represented.
+
+At EGI 2023, we first time held a booth on Galaxy and usegalaxy.eu, presenting the European Galaxy Server to a wider audience from various scientific fields. A dedicated conference session with multiple talks introduced Galaxy and presented its opportunities, as well as the ESG project.
```
Comment (sebastian-schaaf) [inline content/news/2023-07-24-egi2023/index.md:12]:
```suggestion
At EGI2023, we first time held a booth on Galaxy and usegalaxy.eu, presenting the European Galaxy Server to a wider audience from various scientific fields. A dedicated conference session with multiple talks introduced Galaxy and presented its opportunities, as well as the ESG project.
```

File: content/news/2023-07-24-egi2023/index.md [hunk 8]
```diff
@@ -0,0 +1,43 @@
+---
+title: 'Freiburg Galaxy team at the EGI 2023'
+date: "2023-07-24"
+tease: "From June 19th to 23rd, members from the Freiburg Galaxy team traveled to Poznań, Poland to participate and present the EuroScienceGateway (ESG) project at the EGI2023 meeting"
+hide_tease: false
+subsites: [all-eu]
+main_subsite: eu
+---
+
+During 19th-23rd of June, Björn Grüning, Sebastian Schaaf, and Sanjay Kumar Srikakulam from the European Galaxy Team travelled to Poznań to join the EGI2023 conference and presented the [EuroScienceGateway project](https://galaxyproject.org/projects/esg/) along with collaborators. With Sebastián Luna-Valero and Enol Fernández (European Grid Initiative/NL), Hakan Bayındır (ULAKBIM/TR), Łukasz Dutka (Cyfronet/PL), Abdulrahman Azab and Maiken Pedersen (University of Oslo/NO), work packages 1, 3 and 4 of ESG were represented.
```
Comment (sebastian-schaaf) [inline content/news/2023-07-24-egi2023/index.md:10]:
```suggestion
During 19th-23rd of June, Björn Grüning, Sebastian Schaaf, and Sanjay Kumar Srikakulam from the European Galaxy Team travelled to Poznań to join the [EGI2023 conference](https://www.egi.eu/event/egi2023/) and presented the [EuroScienceGateway project](https://galaxyproject.org/projects/esg/) along with collaborators. With Sebastián Luna-Valero and Enol Fernández (European Grid Initiative/NL), Hakan Bayındır (ULAKBIM/TR), Łukasz Dutka (Cyfronet/PL), Abdulrahman Azab and Maiken Pedersen (University of Oslo/NO), work packages 1, 3 and 4 of ESG were represented.
```

File: content/news/2023-07-24-egi2023/index.md [hunk 9]
```diff
@@ -0,0 +1,43 @@
...
+
+The initiative aims to create an open infrastructure for data-driven research, focusing on providing a user-friendly scientific workflow and data integration platform. This endeavor seeks to achieve robustness, scalability, and integration, enabling federated computational infrastructures and democratizing research data analysis.
+
+During the session various aspects of ESG and Galaxy were discussed, including the integration of the OneData platform, exploring [DIRAC ](https://dirac.readthedocs.io/en/latest/index.html) as a new metascheduler, and [ARC ](https://arc-user-guide.readthedocs.io/en/latest/index.html) as a new job-runner.
+
+# Thank You !
+
+The European Galaxy team would like to **thank** the **Galaxy community**, **members of the ESG** for all their contributions that were presented and **EGI** for the fantastic organisation of this meeting.
+
+![EGI 2023 participant picture](https://pbs.twimg.com/media/Fznr0-OWYAEJ85a?format=jpg&name=large)
```
Comment (sebastian-schaaf) [inline content/news/2023-07-24-egi2023/index.md:43]:
```suggestion
![EGI2023 participants picture](https://pbs.twimg.com/media/Fznr0-OWYAEJ85a?format=jpg&name=large)
```

**Review by bgruening — APPROVED** [review]

**Pull request merged**